WalletWasabi.Gui/Converters/FeeDisplayFormatConverter.cs

```diff
@@ -0,0 +1,63 @@
+using Avalonia;
+using Avalonia.Data.Converters;
+using NBitcoin;
+using Splat;
+using System;
+using System.Globalization;
+using WalletWasabi.Exceptions;
+using WalletWasabi.Gui.Models;
+using WalletWasabi.Gui.Controls.WalletExplorer;
+
+namespace WalletWasabi.Gui.Converters
+{
+	public class FeeDisplayFormatConverter : IValueConverter
+	{
+		public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
```

Comment:
It would be nice to add a unit test.

Reply:
Would love to add, there are no current tests for other converters like this. Not familiar enough to go about doing so.

Reply:
Please any question related how to write that test here. 

I would recommend to add two constructors one with no parameters and one constructor for unit testing where you can control parameters passed to `FeeDisplayFormatConverter`.